qiskit_experiments/base_experiment.py

```diff
@@ -99,62 +70,92 @@ def __init__(
                 print(self._num_qubits, self._physical_qubits)
                 raise QiskitError("Duplicate qubits in physical qubits list.")
 
-        # Store options and values
-        self._circuit_options = set(circuit_options) if circuit_options else set()
+        # Experiment options
+        self._experiment_options = self._default_experiment_options()
+        self._transpile_options = self._default_transpile_options()
+        self._run_options = self._default_run_options()
+        self._analysis_options = self._default_analysis_options()
+
+        # Set initial layout from qubits
+        self._transpile_options.initial_layout = self._physical_qubits
 
     def run(
         self,
-        backend: "Backend",
+        backend: Backend,
         analysis: bool = True,
         experiment_data: Optional[ExperimentData] = None,
-        **kwargs,
+        **run_options,
     ) -> ExperimentData:
         """Run an experiment and perform analysis.
 
         Args:
             backend: The backend to run the experiment on.
-            analysis: If True run analysis on experiment data.
-            experiment_data: Optional, add results to existing experiment data.
-                             If None a new ExperimentData object will be returned.
-            kwargs: keyword arguments for self.circuit, qiskit.transpile, and backend.run.
+            analysis: If True run analysis on the experiment data.
+            experiment_data: Optional, add results to existing
+                experiment data. If None a new ExperimentData object will be
+                returned.
+            run_options: backend runtime options used for circuit execution.
```

Comment:
Do you think it is worth adding a link to the [API docs](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.IBMQBackend.html#qiskit.providers.ibmq.IBMQBackend.run)? Of course this is provider agnostic but user cannot find what is run_options with this. You can say something like this is provider dependent but in IBM Quantum you can refer to ...